app/services/title_builder.rb

```diff
@@ -1,20 +1,57 @@
 # frozen_string_literal: true
 
-class TitleBuilder
+class TitleBuilder # rubocop:disable Metrics/ClassLength
```

Comment:
Slight preference for putting this in `rubocop.yml` to avoid rubocruft.

Reply:
Putting this in `.rubocop.yml` means it's something we want to keep exempt from the rules, but in this case we should fix it sometime, or acknowledge that it's a bit of a problem.